src/MobileCoreServices/UTType.cs

```diff
@@ -36,22 +36,64 @@ namespace MobileCoreServices {
 
 	public static partial class UTType {
 
-#if !NET
-		[iOS (8,0)][Mac (10,10)]
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
+		[SupportedOSPlatform ("macos10.10")]
+		[UnsupportedOSPlatform ("tvos14.0")]
```

Comment:
These unsupported come from the parent type [here](https://github.com/tj-devel709/xamarin-macios/blob/acc6c2dd9db7da07db67684129bc106d65aca964/src/mobilecoreservices.cs#L6-L9).